### ISSUE TITLE
launch.json: "envFile" configuration seems to need '.test' suffix amended to the filepath

### ISSUE THREAD

**Issue by lks-dev** [kondratyev-nv/vscode-python-test-adapter#312]
**tl:dr**: `"envFile": "env/file/path.test"` works as expected `"envFile": "env/file/path"` does not, even though there is no version of the file on the filesystem ending with a ".test" suffix. 

On Setting up pytest with an pytest.ini config in the source file, configuring coverage, i needed to a debug launch config setup used by the python-test-adapter extension to add the pytest addopts: '--no-cov' for debugging. The needed environment manipulations in in the .env file even though I had explicitly given the `"envFile":"env/file/path"` as a parameter in the launch config did not take effect.
After some time playing with the example given on the overview page on the marketplace, I added the ".test" suffix to the filepath of my .env file resolving the issue for the me.

As there is no file called `"env/file/path.test"` on my file system i would consider this a bug or, if it is a in some circles popular convention, something worth explaining on the overview section regarding the launch config.

**Comment by lks-dev** [comment]
The actual 'bug' seems not to be related to the test-adapter extension, but to the handling of .env files and the setup of testing and debug environement in general. Using a wrong filepath seemed to result in not (re-?)loading the env file, which resulted in the expected env.
